Lib/test/test_opcache.py

```diff
@@ -714,11 +714,13 @@ def write(items):
         opname = "FOR_ITER_LIST"
         self.assert_races_do_not_crash(opname, get_items, read, write)
 
-    @requires_specialization
+    @requires_specialization_ft
     def test_load_attr_class(self):
         def get_items():
             class C:
-                a = object()
+                # a must be set to an instance that uses deferred reference
+                # counting in free-threaded builds
+                a = type("Foo", (object,), {})
```

Comment:
This looks a bit mysterious to me.  Does this do something different than what `class Foo: [...]` does?  Calling it an instance (which techically true) is a bit confusing too.  I would just say a "set to an object that uses ...".   It's a class object.  If you need it written like this, I think a little helper function with a docstring explaining why it must be done that way would help:
```
def make_deferred_ref_count_obj():
    return type("Foo", (object,), {})
 ```

Reply:
Ah, I see later there is `item.a = type("Foo", (object,), {})` so you want an expression not a statement.